rust/plan.ps1

```diff
@@ -1,12 +1,12 @@
 $pkg_name="rust"
 $pkg_origin="core"
-$pkg_version="1.40.0"
+$pkg_version="1.41.0"
 $pkg_description="Safe, concurrent, practical language"
 $pkg_upstream_url="https://www.rust-lang.org/"
 $pkg_license=@("Apache-2.0", "MIT")
 $pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 $pkg_source="https://static.rust-lang.org/dist/rust-$pkg_version-x86_64-pc-windows-msvc.msi"
-$pkg_shasum="f02d8c31b47916183e0ea1ccb348fa25e39f31867d2a1e0cc212623b5a84047c"
+$pkg_shasum="e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
hmm looks like it is expecting `ad9a80a19de737244f8dfda61506f520ca9a2f8329986fb1f666034da099470f`

Reply:
Whoops, must hashed the wrong file. Should be fixed now.